doc/run_with_hive.md

```diff
@@ -0,0 +1,51 @@
+# Run SQLFlow with Hive via HiveServer2
+
+This document is a tutorial on how to run SQLFlow, which connects to the hive server2.
+
+For the most production environment, the system administrators may setup hive server with [authentication configuration](https://cwiki.apache.org/confluence/display/Hive/Setting+Up+HiveServer2#SettingUpHiveServer2-Authentication/SecurityConfiguration): e.g., KERBEROS, LDAP, PAM, or CUSTOM.
+
+## Connect Hive Server with No SASL
+
+Launch your standalone hive server Docker container by running:
+
+``` bash
+> docker run -d -p 8888:8888 --name=hive sqlflow/gohive:dev python3 -m http.server 8899
+```
+
+This implies settings in `hive-site.xml`:
+
+``` text
+hive.server2.authentication = NOSASL
+```
+
+Test SQLFlow by running a query in Jupyter Notebook
+
+``` bash
+> docker run --rm --net=container:hive sqlflow/sqlflow \
```

Comment:
Do we need to add `-p 8888:8888` so that the Jupyter server in the container can be accessed by the browser on the host?

Reply:
The sqlflow Docker container shares the network stack of hive container by `--net=container:hive ` and the hive container exposed the port to host `-p  8888:8888`.